opencensus/trace/propagation/google_cloud_format.py

```diff
@@ -19,7 +19,7 @@
 from opencensus.trace.trace_options import TraceOptions
 
 _TRACE_CONTEXT_HEADER_NAME = 'X-Cloud-Trace-Context'
-_TRACE_CONTEXT_HEADER_FORMAT = '([0-9a-f]{32})(\/([0-9a-f]{16}))?(;o=(\d+))?'
+_TRACE_CONTEXT_HEADER_FORMAT = r'([0-9a-f]{32})(\/([0-9a-f]{16}))?(;o=(\d+))?'
```

Comment:
This is a separate issue which was caught by the latest pylint.
The string literal used for regular expression was not properly escaped.